bigtop-deploy/juju/hadoop-mapreduce/README.md

```diff
@@ -50,8 +50,13 @@ units. To deploy this bundle, simply use:
 
     juju quickstart bigtop-processing-mapreduce
 
```

Comment:
Is the bundle name "bigtop-processing-mapreduce" or "hadoop-mapreduce?"  I would prefer a shorter name, and we may not need to include "bigtop" in the name as that is already in the path.  I think there was a comment on the list to exchange "mapreduce" for "yarn." I am not partial to either of these, I think they are both fair names.  I do think mapreduce does give a better indicator what the end objective is though.